src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -916,17 +908,17 @@ private List<Field> GetNodeOrNodeListFields(TreeType node)
 
         private void WriteRedType(TreeType node)
         {
-            WriteComment(node.TypeComment, "  ");
+            WriteComment(node.TypeComment, "");
 
             if (node is AbstractNode)
             {
-                AbstractNode nd = (AbstractNode)node;
-                WriteLine("  public abstract partial class {0} : {1}", node.Name, node.Base);
-                WriteLine("  {");
-                WriteLine("    internal {0}(Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax.CSharpSyntaxNode green, SyntaxNode parent, int position)", node.Name);
```

Comment:
we're already in the `Microsoft.CodeAnalysis.CSharp.Syntax` namespace.  So this could be abbreviated.  This also makes us more consistent with the rest of the file (where we do this already).